src/main/java/teammates/ui/output/AccountRequestCreateErrorResults.java

```diff
@@ -0,0 +1,63 @@
+package teammates.ui.output;
+
+/**
+ * The API output format of the error results when account request create request fails.
+ */
+public class AccountRequestCreateErrorResults extends ApiOutput {
```

Comment:
1. Frontend is not implemented to check more complex errors and for those it relies on backend
2. I think you understand why the error messages are returned like this. It's for frontend to show the error message for a specific field under the corresponding inpit box instead of combining everything together. The old approach to combine everything together negatively impacts user experience dramatically, especially for a form with many fields.